x-pack/plugins/security_solution/public/common/components/event_details/event_fields_browser.tsx

```diff
@@ -4,9 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { sortBy } from 'lodash';
 import { EuiInMemoryTable } from '@elastic/eui';
 import React, { useCallback, useMemo } from 'react';
+
```

Comment:
is blank line here necessary? and I think we try to keep lodash imports at the top